temporal-opentracing/src/main/java/io/temporal/opentracing/OpenTracingConfig.java

```diff
@@ -0,0 +1,125 @@
+/*
+ *  Copyright (C) 2020 Temporal Technologies, Inc. All Rights Reserved.
+ *
+ *  Copyright 2012-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Modifications copyright (C) 2017 Uber Technologies, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not
+ *  use this file except in compliance with the License. A copy of the License is
+ *  located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed on
+ *  an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+package io.temporal.opentracing;
+
+public class OpenTracingConfig {
+  private final String workflowStartOperationNamePrefix;
```

Comment:
I would put these into the map and make key an enum (e.g. TracingNamePrefix with options like Start, StartWithSignal, etc.). This should make code a bit cleaner both here and on the consumer end.

Reply:
Good idea, implemented.